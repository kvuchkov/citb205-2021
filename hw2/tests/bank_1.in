show AAA123
show BBB321
show CCC999
show Alan Fiction
show Bob Doe
show Chad B. Testings
wire 500 AAA123 CCC999
show AAA123
show BBB321
show CCC999
wire 1500 AAA123 CCC999
wire 1500 AAA123 CCC99
wire 1500 AAA12 CCC999
show AAA123
show CCC999
wire 3500 CCC999 BBB321
show CCC999
show BBB321