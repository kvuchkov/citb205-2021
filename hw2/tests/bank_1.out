Accounts loaded
AAA123 (Alan Fiction): $1000.00
BBB321 (Bob Doe): $4000.00
CCC999 (Chad B. Testings): $3000.00
AAA123 (Alan Fiction): $1000.00
BBB321 (Bob Doe): $4000.00
CCC999 (Chad B. Testings): $3000.00
500.00 was wired successfully
AAA123 (Alan Fiction): $500.00
BBB321 (Bob Doe): $4000.00
CCC999 (Chad B. Testings): $3500.00
Cannot wire amount: insufficient funds
Cannot wire amount: cannot find destination account
Cannot wire amount: cannot find source account
AAA123 (Alan Fiction): $500.00
CCC999 (Chad B. Testings): $3500.00
3500.00 was wired successfully
CCC999 (Chad B. Testings): $0.00
BBB321 (Bob Doe): $7500.00
